src/figuresClasses.ts

```diff
@@ -1,19 +1,109 @@
-export interface Figure {
 
+export interface Figure {
+  color: 'red' | 'green' | 'blue';
+  shape: string;
+  radius?: number;
```

Comment:
In Figure interface you should add only those properties, that should be in each figure
Add radius only for Circle, not for each Figure